changelog/29677.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+ui: add KV view for wrap tool
```

Comment:
Thanks for including a changelog! 😍 

"KV" can be a little vague and sounds like it references the KV secrets engine. Can we reword to be more explicit? Something like "adds key value pair string inputs as optional form for wrap tool"